CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # master
 *Please add new entries at the top.*
+1. Renamed `filterMap` to `compactMap` and deperacated `filterMap` (#746, kudos to @Marcocanc)
```

Comment:
_deprecated_ is misspelled here

Reply:
Oops, good catch. At least fixing that will retrigger the build :)